strict-base-types/strict-base-types.cabal

```diff
@@ -129,8 +129,8 @@ library
   build-depends:
       base       >= 4.5     && < 5
     , lens       >= 4.19.2  && < 4.20
-    , QuickCheck >= 2.14.1  && < 2.15
     , aeson      >= 1.4.6.0 && < 1.6
+    , quickcheck-instances >= 0.3.24 && <0.4
```

Comment:
This pulls in QuickCheck as a dependency. I had thought it would be better to define our instances there, people (including myself) are more relaxed about having lots of test dependencies.

Reply:
`QuickCheck` won't depend on anything externally. Plenty of instances (even for some types in `base`) live in `quickcheck-instances`.

Reply:
Misread the PR originally, I thought this was a change to `strict`. So this is just backwards-compat for strict-base-types.